crowdin/messages.json

```diff
@@ -155,7 +155,6 @@
   "236080779": "Feature-rich charts",
   "236686595": "24-hour trading (Monday-Friday), based on forex market hours",
```

Comment:
You missed these files:

- pages/markets/basket-indices/index.tsx
- pages/markets/commodities/index.tsx
- pages/markets/components/helper/_available-options.tsx
- pages/markets/components/helper/_available-platforms.tsx